spec/controllers/reviews_controller_spec.rb

```diff
@@ -0,0 +1,120 @@
+require 'rails_helper'
+
+RSpec.describe ReviewsController, type: :controller do
+  describe 'POST /create' do
+    subject { post :create, params: }
+
+    let(:book) { create(:book) }
+    let(:params) { { book_id: book.id, review: { comment: 'test review controller', star: 4 } } }
+
+    it 'creates review success' do
+      expect(subject.status).to eq(302)
+      expect(Review.count).to eq(1)
+
+      # test value of each attributes
+      created_review = assigns(:review)
+      expect(Review.find(created_review.id).comment).to eq('test review controller')
+      expect(Review.find(created_review.id).star).to eq(4)
+      expect(Review.find(created_review.id).book).to eq(book)
+
+      # redirect to show page
+      expect(subject).to redirect_to(book_path(created_review.book))
+    end
+
+    context 'validation' do
+      let(:params) { { book_id: book.id, review: { comment: '' } } }
+
+      it 'requires comment' do
+        # subject
+        expect(subject.status).to eq(302)
+        expect(assigns(:review)).not_to be_valid
+        expect(Review.count).to eq(0)
+      end
```

Comment:
```suggestion
      context 'when comment is invalid' do
        let(:params) { { book_id: book.id, review: { comment: '', star: 4 } } }

        it 'not create new review' do
          expect(subject.status).to eq(302)
          expect(assigns(:review)).not_to be_valid
          expect(Review.count).to eq(0)
        end
      end
```

Maybe this look better?
I added a context because don't want to define params twice in 1 block.
Also add valid `star` to params to make sure that missing star will not make API error.

Let's do this for next `it` too